### PR TITLE
Dcp 552 Test downloaded links

### PR DESCRIPTION
**Pull request by aaclan-ebi**
ebi-ait/dcp-ingest-central#552

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -51,6 +51,7 @@ test_bulk_update:
   script:
      - python -m unittest tests.test_ingest.TestRun.test_bulk_update
   only:
+    - dcp-552-int-test-download-links
```
Comment (ke4) [inline .gitlab-ci.yml]: Are these 2 changes in this config file intentional?

File: tests/ingest_agents.py [hunk 2]
```diff
@@ -168,6 +168,12 @@ def get_biomaterials(self):
         def get_bundle_manifests(self):
             return self._get_entity_list('bundleManifests')
 
+        def get_submission_manifest(self):
+            return self._get_entity('submissionManifest')
```
Comment (ke4) [inline tests/ingest_agents.py:172]: This one and line 175 will throw an exception when `self._get_entity` returns `None`,

**Review by ke4 — CHANGES_REQUESTED** [review]
Added 2 comments. Could you review them please? Tx

File: tests/ingest_agents.py (hunk 2 above)
Comment (aaclan-ebi) [reply tests/ingest_agents.py:172]: why?  'submissionManifest' and 'summary' are parameters to _get_entity

File: .gitlab-ci.yml (hunk 1 above)
Comment (aaclan-ebi) [reply .gitlab-ci.yml]: no, i will revert them now, did that to test it in gitlab

File: tests/ingest_agents.py (hunk 2 above)
Comment (ke4) [reply tests/ingest_agents.py:172]: Ok, I misread the `(` to a `[`. Sorry. Should have gone to Specsavers. :-D

**Review by ke4 — APPROVED** [review]
LGTM

**Review by jacobwindsor — APPROVED** [review]

**Pull request merged**